graph/resolver/schema.resolvers.go

```diff
@@ -489,6 +489,40 @@ func (r *mutationResolver) InviteUserToDiscussion(ctx context.Context, discussio
 	return r.DAOManager.InviteUserToDiscussion(ctx, userID, discussionID, invitingParticipantID)
 }
 
+func (r *mutationResolver) InviteTwitterUsersToDiscussion(ctx context.Context, discussionID string, twitterUsers []*model.TwitterUserInput, invitingParticipantID string) ([]*model.DiscussionInvite, error) {
+	authedUser := auth.GetAuthedUser(ctx)
+	if authedUser == nil {
+		return nil, fmt.Errorf("Need auth")
+	}
+
+	participantResponse, err := r.DAOManager.GetParticipantsByDiscussionIDUserID(ctx, discussionID, authedUser.UserID)
+	if err != nil {
+		return nil, err
+	}
+	if participantResponse == nil {
+		return nil, fmt.Errorf("Failed to find participant with ID %s", invitingParticipantID)
+	}
+
+	// Verify that the updating participant belongs to the logged-in user
+	var nonAnonUserID, anonUserID string
+	if participantResponse.NonAnon != nil && participantResponse.NonAnon.ID == invitingParticipantID {
+		nonAnonUserID = *participantResponse.NonAnon.UserID
+	}
+	if participantResponse.Anon != nil && participantResponse.Anon.ID == invitingParticipantID {
+		anonUserID = *participantResponse.Anon.UserID
+	}
+	if authedUser.UserID != nonAnonUserID && authedUser.UserID != anonUserID {
+		return nil, fmt.Errorf("Unauthorized")
+	}
+
+	client, err := r.DAOManager.GetTwitterClient(ctx)
```

Comment:
Can we rename this function to `GetTwitterClientWithUserTokens` or something a bit more clear that it's using the requesting user's oauth tokens?

Reply:
Yes, this will be addressed in the next commit